website/src/scss/parts/_container.scss

```diff
@@ -0,0 +1,94 @@
+$container: (
+  container: (
+    mobile: (
+      padding-left: 16px,
+    ),
+    desktop: (
+      padding-left: 85px,
+    ),
+  ),
+);
+
+.container {
+  @include responsive($container, container);
+}
+
+.docTitle_node_modules-\@docusaurus-theme-classic-src-theme-DocItem- {
+  font-size: 32px;
+  margin-bottom: 20px;
+}
+
+.docItemContainer_node_modules-\@docusaurus-theme-classic-src-theme-DocItem- article {
+  max-width: 640px;
+}
+
+code {
+  background-color: $base-white-color;
+}
+
+blockquote {
+  border-left-color: $base-mandy-color;
+  background-color: $base-citrin-color;
+  color: $base-cape-color;
+}
+
+blockquote code {
+  font-family: $base-code-block-family;
+  background-color: $base-chamois-color;
+  color: $base-cape-color;
+}
+
+iframe {
+  width: 100%;
+  height: 100vh;
+}
+
+/* Right sidemenu */
+.contents.contents__left-border {
+  border-left-color: $base-silver-color;
+  padding-left: 22px;
+}
+
+.contents .contents__link {
+  color: $base-cape-color;
+  font-size: 12px;
+  line-height: 2;
+}
+
+.contents .contents__link:hover,
+.contents .contents__link:hover code,
+.contents .contents__link.contents__link--active,
+.contents .contents__link.contents__link--active code {
+  color: $base-mandy-color;
+}
```

Comment:
This needs to be rewritten in the sass way...

Reply:
The issue with some CSS rules is that Docusaurus has its own styles and when I try to overwrite them the sass way they don't get overwritten because of the priority.

I could solve this by adding `!important` to these rules. What do you think?

Reply:
things like `.contents .contents__link:hover` should be written like

```scss
.contents {
  & &__link {
    &:hover {
    ...
    }
  }
}

```

Should still compile like the above and should work, but will look more structured.

Reply:
This should be enough:

```scss
.contents {
  &__link {
    &:hover {
    ...
    }
  }
}
```

Reply:
Yeah, unless he has some weird specificity issues (like he mentioned) 😄

Reply:
This was a typo. It will be fixed. :)